DeepSpeech.py

```diff
@@ -963,17 +962,10 @@ def done(self):
                     self.samples.sort(key=lambda s: s.wer)
 
                     # Append WER to WER log file
-                    if len(FLAGS.wer_log_file) > 0:
-                        hash = get_git_revision_hash()
+                    if len(FLAGS.wer_log_pattern) > 0:
                         time = datetime.datetime.utcnow().isoformat()
-                        # Append to log file
-                        with open(FLAGS.wer_log_file, 'a') as wer_log_file:
-                            if wer_log_file.tell() > 0:
-                                wer_log_file.write('\n')
-                            wer_log_file.write('logwer("%s", "%s", "%s", %f)' % (hash, time, self.set_name, self.wer))
-                        # Publish to web server
-                        if FLAGS.publish_wer_log:
-                            maybe_publish()
```

Comment:
For the moment I'd like to keep `maybe_publish`, as I don't know, if I'll use it later for publishing to http://doyouunderstand.me ...